Core/include/Acts/Utilities/MathHelpers.hpp

```diff
@@ -12,6 +12,50 @@
 
 namespace Acts {
 
+/// @brief Returns the absolute of a number
+///        (Can be removed for c++ 23)
+template <typename T>
+constexpr T abs(const T n) {
+  if constexpr (std::is_signed_v<T>) {
+    if (n < 0) {
+      return -n;
+    }
+  }
+  return n;
+}
+
+/// @brief Calculates the ordinary power of the number x.
+/// @param x: Number to take the power from
+/// @param p: Power to take
+template <typename T, std::integral P>
```

Comment:
Is the goal for this to be constexpr and? If the exponents aren't to large why not a simpler recursive function like this:

```cpp
template <typename T, std::integral P>
constexpr T pow(T base, P exp) {
    return exp == 0 ? 1 : base * pow_constexpr(base, exp - 1);
}
```